.travis.yml

```diff
@@ -19,14 +19,32 @@ env:
         - COASTLINEDIR="$HOME/gmt-install-dir/coast"
         - PATH="$INSTALLDIR/bin:$PATH"
         - LD_LIBRARY_PATH="$INSTALLDIR/lib:$LD_LIBRARY_PATH"
+        - TRAVIS_EVENT_TYPE=cron
+        - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+            export PYTHON=3.6
+            export CONDA_REQUIREMENTS="ci/requirements.txt"
+            export BUILD_DOCS=true
```

Comment:
```suggestion

```

Probably no need for this variable as well since we're only building on cron jobs.